net/net.go

```diff
@@ -204,6 +204,9 @@ func ServiceConnections(listener net.Listener, p2id participant2ID, l Logger) (<
 		for atomic.LoadUint32(&stopFlag) == 0 {
 			conn, err := listener.Accept()
 			if err != nil {
+				if ne, ok := err.(net.Error); ok && ne.Temporary() {
+					continue
+				}
```

Comment:
Good catch. 
Not sure if this is the right approach as `Temporary()` is deprecated. What about checking if the error is a timeout instead, wait some short time and try again. WDYT?

Reply:
Hi, @mbrandenburger , updated the PR, use `ne.Timeout` and wait a short time on accept failure.  The logic of wait time is based on the implementation in [net/http](https://cs.opensource.google/go/go/+/refs/tags/go1.23.5:src/net/http/server.go;l=3336-3345).